quickcheck-dynamic/src/Test/QuickCheck/StateModel.hs

```diff
@@ -325,30 +328,30 @@ instance Eq (Step state) where
 -- We also collect a list of names of actions which were generated,
 -- but were then rejected by their precondition.
 
-data Actions state = Actions_ [String] (Smart [Step state])
+data Actions state = Actions_ [String] state (Smart [Step state])
```

Comment:
At this stage, a record would help understand what are the various parts of the data type